### DISCUSSION TITLE
Make the `edit this page` button more useful

### DISCUSSION THREAD

**Discussion by hugo-sid** [colinwilson/lotusdocs#37] [Feature Requests]
## Current behavior

Clicking on the 'edit this page' button opens the corresponding Markdown file from the most recent commit.

### What's the problem

The edit this file option on GitHub is disabled, when it is opened in the above mentioned manner. So, the purpose (allowing one to edit) is not served.

![image](https://github.com/colinwilson/lotusdocs/assets/122173059/3b5f2382-e917-46c3-b0db-c10f3d472ba1)

## Expected behavior with example

Consider the URL: https://lotusdocs.dev/docs/guides/features/feedback-widget/
Clicking on the 'edit this page' button opens: https://github.com/colinwilson/lotusdocs.dev/blob/9907176ffb06a0c8652386aa9f72668a90d85bdc/content/docs/guides/features/feedback-widget.md/ - edit option is disabled.

It would be better if the 'edit this page' button opens any of the following links:

- Kind 1 (blob link): https://github.com/colinwilson/lotusdocs.dev/blob/release/content/docs/guides/features/feedback-widget.md
- Kind 2 (edit link): https://github.com/colinwilson/lotusdocs.dev/edit/release/content/docs/guides/features/feedback-widget.md

### Example of Kind 1 link

https://gohugo.io/content-management/organization/#url

![image](https://github.com/colinwilson/lotusdocs/assets/122173059/9d6c3390-750b-40b1-ae6d-2dbd076201fa)

### Example of Kind 2 link

https://developers.cloudflare.com/workers/platform/limits#log-size

![image](https://github.com/colinwilson/lotusdocs/assets/122173059/27ba9792-5790-4273-93fb-d5fc980ff425)

## Solution

One possible solution (not tested) can be to use:

```go
<a href="{{.Site.Params.ghrepo}}edit/release/content/{{ .Lang }}/{{.File.Path}}">Edit this page</a>
```

Then in `config/_default/params.toml` :

```toml
## GH repo include trailing slash
ghrepo = "https://github.com/colinwilson/lotusdocs/"
```

**Comment by colinwilson** [comment]
Thank you! 👍

I hadn't noticed the edit was disabled. I'll fix this on the next update.

**Reply by hugo-sid** [reply]
> Thank you! 👍

Thank you for the beautiful theme @colinwilson.

> I'll fix this on the next update.

Great.

**Reply by colinwilson** [reply]
Fixed by 4f16b31 and 6bc7970
